components/org.wso2.carbon.identity.conditional.auth.functions.notification/src/test/java/org/wso2/carbon/identity/conditional/auth/functions/notification/SendEmailFunctionImplTest.java

```diff
@@ -58,6 +59,7 @@ protected void setUp() throws Exception {
 
         super.setUp();
 
+        CarbonConstants.ENABLE_LEGACY_AUTHZ_RUNTIME = true;
```

Comment:
what is the usecase behind this?

Reply:
This is used in the pre test set ups. However there is no default value provided for this in CarbonConstants. Therefore this gives a NullPointerException when tests are run. Therefore the value was initialised here before tests run.